viewing_party/party.py

```diff
@@ -1,7 +1,7 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie = []

```

Comment:
what if we turned this into a dictionary?

```suggestion
    movie = { }
```